website/content/docs/concepts/billing-start-date.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: Billing start date
+description: >-
+   A description of Vault cluster's billing start date used throughout Vault.
+---
+
+# Billing start date
+
+This is the billing start date associated with the reporting cluster. This will be the license start date if not configured.
+
+As of 1.15.13, 1.16.7, 1.17.3, 1.18+, Vault clusters will automatically adjust the billing start date to provide a year's worth of utilization data.
```

Comment:
```suggestion
As of 1.15.13, 1.16.7, 1.17.3, 1.18+, Vault clusters will automatically adjust the billing start date to provide a year's worth of utilization data.
```
```suggestion
To provide a one-year window of utilization data, Vault clusters automatically adjust the billing start date [to | by] {DESCRIPTION_OF_ADJUSTMENT}.
```
I'm still not clear exactly *how* we adjust the billing start date (hence the "variable" in the suggestion.

As a general note, except when we're calling out bugs/important changes in things like release notes or changelogs, docs are versions to specific Vault releases. The docs in `main` should always focus on the the current Vault version rather than calling out multiple versions. 

Since we've limited backport behavior to the most current version of Vault, that means we need to directly update the docs for any other versions with separate PRs (either by directly creating a new PR or manually backporting changes in another PR).